change/office-ui-fabric-react-2020-02-03-10-40-32-fix-datetime-test.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
```

Comment:
The change type can be `none` since it doesn't affect published package.. just tests

Reply:
Oooh okay, I'll change that